test/10.js

```diff
@@ -0,0 +1,20 @@
+const solution = require('../solutions/10');
+
+const test = (testNum1, testNum2) => {
+  let count = 0;
+  let compare = true;
+  const testFunc = (x) => {
+    count += 1;
+    if (count !== x) compare = false;
+  };
+  solution(testFunc, testNum1, testNum2);
+  if (compare && (count == testNum1 || count == testNum2)) {
```

Comment:
if 5, 10 is passed in and your function is only executed 5 times, then you are console logging correct even though it is not.